agents/PAW-02B Impl Planner.agent.md

```diff
@@ -279,25 +284,45 @@ After writing the plan to `.paw/work/<feature-slug>/ImplementationPlan.md`:
    **IF Review Strategy = 'prs'**:
    - Ensure on planning branch: `git branch --show-current`, create if needed: `git checkout -b <target_branch>_plan`
    - Stage artifacts: `git add .paw/work/<feature-slug>/{Spec.md,SpecResearch.md,CodeResearch.md,ImplementationPlan.md}` and prompt files
-   - Verify: `git diff --cached`, commit, push: `git push -u <remote> <target_branch>_plan`
+   - Verify: `git diff --cached`
+   - Commit following structured Commit Message Format:
+     - Extract Work Title and Work ID from WorkflowContext.md
+     - Format: `[<Work Title>] <type>: <description>` (omit Phase N for planning work)
+     - Use type `docs:` or `chore:` for planning artifacts
+     - Add PAW footer: `PAW-Phase: none` and `Work-ID: <feature-slug>`
+     - Example: `[Auth System] docs: create implementation plan with 3 phases`
+   - Push: `git push -u <remote> <target_branch>_plan`
    - Create Planning PR (`<target_branch>_plan` → `<target_branch>`):
      - Title: `[<Work Title>] Planning: <brief description>`
      - Body format:
        ```
        **🐾 Implementation Planner 🤖:**
        
+       Related to #<N>
+       
        [Summary of deliverables, links to artifacts]
        
        ---
        🐾 Generated with [PAW](https://github.com/lossyrob/phased-agent-workflow)
        ```
+     - Extract issue number from WorkflowContext.md's Issue URL field:
+       - If Issue URL is "none": Use "No associated issue" instead of "Related to #N"
+       - If Issue URL provided: Extract number from GitHub URL format `https://github.com/<owner>/<repo>/issues/<N>` → use `#<N>`
+       - Examples: `https://github.com/lossyrob/phased-agent-workflow/issues/59` → `#59`
+     - This creates bidirectional link between PR and issue in GitHub timeline
```

Comment:
This is too many tokens to spend on this. Just a one liner is fine. And it doesn't pull from the WorkflowContext.md, it gets it from get_paw_context. You don't have to mention the source it will know where to get it.